web/src/components/toolbar/ColorerOption.vue

```diff
@@ -0,0 +1,67 @@
+<script>
+export default {
+  props: {
+    title: {
+      type: String,
+      required: false,
+      default: '',
+    },
+    value: {
+      type: String,
+      required: false,
+      default: null,
+    },
+    disabled: {
+      type: Boolean,
+      required: false,
+    },
+    options: { // {name: string, options: {name: string, color?: string}[]}[]
+      type: Array,
+      required: true,
+    },
+  },
+  computed: {
+    showSelect() {
+      return !this.value || this.options.length > 1;
+    },
+    filterOptions() {
+      const selected = this.options.find(d => d.value === this.value);
+      return selected ? selected.options : [];
+    },
+    hasOptions() {
+      if (this.options.length === 0) {
+        return false;
+      }
+      if (this.options.length === 1 && !this.options[0].value) {
+        return false;
+      }
+      return true;
+    },
+  },
+};
+</script>
+
+<template lang="pug">
+div(v-if="hasOptions")
+  v-toolbar.darken-3(color="primary", dark, flat, dense, :card="false")
+    v-toolbar-title
+      slot(name=title) {{title}}
```

Comment:
Can this slot be removed?  I does not appear to ever get used.   Otherwise, please quote attribute value.